templates/docs/examples/patterns/navigation/_full-width-dropdown.jinja

```diff
@@ -0,0 +1,87 @@
+{% macro dropdown(id, title, is_open=False) %}
+<div class="p-navigation__dropdown is-full-width {% if not is_open %}is-collapsed{% endif %}"  data-level="1" id="{{ id }}" {% if not is_open %}aria-hidden="true"{% endif %}>
+  <div class="p-navigation__dropdown-content-mobile">
+    <ul class="p-list u-no-margin">{# TODO: use p-list temporarly, we will later have more specific styling class name for it #}
+      <li class="p-navigation__item--dropdown-close" id="{{ id }}-back">
+        <button aria-controls="{{ id }}" class="p-navigation__link js-back">
```

Comment:
```suggestion
        <button aria-controls="{{ id }}" class="p-navigation__link js-back-button">
```
Could we make this 'js-back-button' to align with 'js-menu-button' and 'js-search-button'?

Reply:
Sure, I haven't been touching much of the sliding nav things, but it doesn't hurt to do small fixes like this right away.

Reply:
Done.